src/proto/graplinc/common/v1beta1/types.proto

```diff
@@ -11,3 +11,30 @@ message Uuid {
   // Most significant bytes
   fixed64 msb = 2;
 }
+
```

Comment:
Can we add a comment about why we're rolling our own time primitives? (I'm not questioning your motives, I just want future users to know what's up)

Reply:
done